portal-frontend/src/app/features/alcs-edit-submission/alcs-edit-submission.component.ts

```diff
@@ -189,6 +191,9 @@ export class AlcsEditSubmissionComponent implements OnInit, OnDestroy, AfterView
         if (this.profoProposalComponent) {
```

Comment:
looks like a good time to move EditApplicationSteps.Proposal into separate function to make it easier to read the code

Reply:
Good call, done!